api/types/server.go

```diff
@@ -335,7 +334,18 @@ func (s *ServerV2) CheckAndSetDefaults() error {
 
 // DeepCopy creates a clone of this server value
 func (s *ServerV2) DeepCopy() Server {
-	return proto.Clone(s).(*ServerV2)
+	newServer := *s
```

Comment:
Potential drawback is this needs attention if the server type is modified to include more pointer fields that would require cloning.

Reply:
Reverted to `proto.Clone` provided that timestamps are all UTC.